TestCPP11.cmake

```diff
@@ -98,6 +100,13 @@ while(TESTS_CPP11)
 
   if(CXX_${TEST_CPP11_NAME}_SUPPORTED)
     add_definitions(-DCXX_${TEST_CPP11_NAME}_SUPPORTED)
+    list(APPEND __supported ${TEST_CPP11_name})
+  else()
+    list(APPEND __unsupported ${TEST_CPP11_name})
   endif()
 endwhile()
 
+if(__unsupported)
+  set(__unsupported " UNSUPPORTED: ${__unsupported}")
+endif()
+message(STATUS "Tested C++11 features: ${__supported}${__unsupported}")
```

Comment:
I hadn't realised I was reintroducing the same thing you had removed... It prints only one line at the top of the cmake run, but if you don't like it we can think of something else to warn users if this executes multiple times. A message only if CMAKE_SOURCE_DIR != PROJECT_SOURCE_DIR for instance?